src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -197,29 +196,22 @@ void CouplingSchemeConfiguration::xmlTagCallback(
                   _config.maxTimeWindows);
   } else if (tag.getName() == TAG_TIME_WINDOW_SIZE) {
     _config.timeWindowSize = tag.getDoubleAttributeValue(ATTR_VALUE);
-    _config.validDigits    = tag.getIntAttributeValue(ATTR_VALID_DIGITS);
-    PRECICE_CHECK((_config.validDigits >= 1) && (_config.validDigits < 17), "Valid digits of time window size has to be between 1 and 16.");
     // Attribute does not exist for parallel coupling schemes as it is always fixed.
     _config.dtMethod = getTimesteppingMethod(tag.getStringAttributeValue(ATTR_METHOD, VALUE_FIXED));
     if (_config.dtMethod == constants::TimesteppingMethod::FIXED_TIME_WINDOW_SIZE) {
-      PRECICE_CHECK(_config.timeWindowSize > 0,
-                    "Time window size has to be larger than zero. "
-                    "Please check the <time-window-size value=\"{}\" valid-digits=\"{}\" method=\"{}\" /> tag "
-                    "in the <coupling-scheme:...> of your precice-config.xml",
-                    _config.timeWindowSize, _config.validDigits, tag.getStringAttributeValue(ATTR_METHOD));
+      PRECICE_CHECK(_config.timeWindowSize > _minTimeStepSize,
```

Comment:
```suggestion
      PRECICE_CHECK(math::greaterEquals(_config.timeWindowSize, _minTimeStepSize),
```
Should be >=. A `_minTimeStepSize` that is identical to the `timeWindowSize` is on the borderline, but should be fine.

Reply:
`_minTimeStepSize` is guaranteed to be greater zero? Might still not harm to put a `PRECICE_ASSERT(math::greater(_minTimeStepSize, 0))` here to make this assumption explicit. I'm so picky here, because previously it was `PRECICE_CHECK(_config.timeWindowSize > 0, ...)` and we don't want to run into the corner case that `timeWindowSize = 0` is allowed.

Reply:
Yes, _minTimeStepSize has to be greater than 0, since we otherwise run into troubles. I would also avoid using math::greaterEquals here, since they contain a tolerance of NUMERICAL_ZERO_DIFFERENCE which might be troublesome.